nodejs12.x/cookiecutter-typescript-app-template/{{cookiecutter.project_name}}/src/common/apigateway/apigateway-event.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiGatewayRequestContext } from './apigateway-request-context';
+
+export interface ApiGatewayEvent {
```

Comment:
I know there are TypeScript definitions in the `aws-sdk` but I couldn't locate that the API Gateway event had a definition that I could use. If it exists please share what it is and I can delete my interfaces and use those included with `aws-sdk`.